Lib/test/test_site.py

```diff
@@ -570,6 +570,8 @@ def _create_underpth_exe(self, lines, exe_pth=True):
             dll_file = os.path.join(temp_dir, os.path.split(dll_src_file)[1])
             shutil.copy(sys.executable, exe_file)
             shutil.copy(dll_src_file, dll_file)
+            for fn in glob.glob(os.path.join(os.path.split(dll_src_file)[0], "vcruntime*.dll")):
```

Comment:
Probably, pathlib would be more readable here.

```suggestion
            for fn in Path(dll_src_file).parent.glob('vcruntime*.dll'):
```